torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -5761,10 +5761,10 @@ def forward(self, _):
         )
         def test_broadcast_object_list(self):
             # Only set device for NCCL backend since it must use GPUs.
+            # Case where rank != GPU device.
+            next_rank = (self.rank + 1) % int(self.world_size)
```

Comment:
curious, is `int()` here necessary?

Reply:
self.world_size is typed str.  thus int is needed.